src/sonic-bgpcfgd/bgpcfgd/managers_rm.py

```diff
@@ -72,7 +73,16 @@ def __del_handler_validate(self, key):
         return True
 
     def __update_rm(self, rm, data):
-        cmds = ["route-map %s permit 100" % ("%s_RM" % rm), " set community %s" % data["community_id"]]
-        log_debug("BGPRouteMapMgr:: update route-map %s community %s" % ("%s_RM" % rm, data["community_id"]))
-        self.cfg_mgr.push_list(cmds)
+        cmds = []
+        if rm is "FROM_SDN_SLB_ROUTES":
+            cmds.append("route-map %s permit 100" % ("%s_RM" % rm))
+            bgp_asn = \
+                self.directory.get_slot("CONFIG_DB", swsscommon.CFG_DEVICE_METADATA_TABLE_NAME)['localhost']['bgp_asn']
+            cmds.append(" set as-path prepend %s %s" % (bgp_asn, bgp_asn))
```

Comment:
Prefer to have a fixed value for AS prepend instead of prepending the current device's AS value. This could be gotten from /etc/sonic/constants.yml : 2. Not sure if this is significant changes.

Reply:
updated.